modelmatrix-core/src/main/scala/com/collective/modelmatrix/db/DatabaseConfig.scala

```diff
@@ -0,0 +1,56 @@
+package com.collective.modelmatrix.db
+
+import com.typesafe.config.{Config, ConfigFactory}
+import slick.driver.{H2Driver, JdbcProfile, PostgresDriver}
+
+/**
+ * Created by jpocalan on 7/27/15.
+ */
+// TODO: replace by some sort of enum
```

Comment:
"Created by" comment becomes useless too fast in shared codebase environment, git blame/idea annotation answers code author question much better. You can remove it somewhere in Idea properties